contractcourt/channel_arbitrator.go

```diff
@@ -1093,7 +1093,7 @@ func (c *ChannelArbitrator) stateStep(
 		// Now that we know we'll need to act, we'll process the htlc
```

Comment:
maybe also change the comment here as well because not only htlc resolvers will be launched?